src/question_types.ts

```diff
@@ -20,6 +20,7 @@ type PartialBlindTestQuestion = PartialClearAnswerQuestion & {
     type: 'BlindTestQuestion';
     path: string;
     answerImage: string;
+    tag?: string;
```

Comment:
Can we rename this to something that explains a bit more what it does, especially as we already have a "tags" property for question selection + From my understanding it is related to which audio filter will be used?

Also, is there a set of valid values for the fielf?
If so, please replace "string" by the set of values, like `"2x" | "reversed" | "underwater"`